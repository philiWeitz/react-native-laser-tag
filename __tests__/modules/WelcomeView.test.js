/* global describe, it, expect */

import React from 'react';
import { shallow } from 'enzyme';
import enzymeToJson from 'enzyme-to-json';
import configureStore from 'redux-mock-store';

import WelcomeViewConnected from '../../src/modules/welcome/WelcomeView';


const mockStore = configureStore([]);

const navProp = {
  navigation: {},
};


describe('welcome-view-connected-tests', () => {

  it('shallow welcome view snapshot test', () => {
    const wrapper = shallow(
      <WelcomeViewConnected navigation={navProp} />,
      { context: { store: mockStore({}) } },
    );
    expect(enzymeToJson(wrapper.dive())).toMatchSnapshot();
  });

});
