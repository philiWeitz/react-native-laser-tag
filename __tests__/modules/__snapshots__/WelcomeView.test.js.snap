// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`welcome-view-connected-tests shallow welcome view snapshot test 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "padding": 20,
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <Button
      containerStyle={
        Object {
          "marginHorizontal": 20,
          "marginVertical": 20,
        }
      }
      onPress={[Function]}
      text=""
    />
  </View>
</View>
`;
